pySDC/implementations/problem_classes/HeatEquation_1D_FEniCS_matrix_forced.py

```diff
@@ -9,29 +9,74 @@
 
 # noinspection PyUnusedLocal
 class fenics_heat(ptype):
-    """
-    Example implementing the forced 1D heat equation with Dirichlet-0 BC in [0,1]
-
-    Attributes:
-        V: function space
-        M: mass matrix for FEM
-        K: stiffness matrix incl. diffusion coefficient (and correct sign)
-        g: forcing term
-        bc: boundary conditions
+    r"""
+    Example implementing the forced one-dimensional heat equation with Dirichlet boundary conditions
+
+    .. math::
+        \frac{d u}{d t} = \nu \frac{d^2 u}{d x^2} + f
+
+    for :math:`x \in \Omega:=[0,1]`, where the forcing term :math:`f` is defined by
+
+    .. math::
+        f(x, t) = -\cos(\pi * x) (\sin(t) - \nu \pi^2 \cos(t)).
+
+    The exact solution of the problem is
+
+    .. math::
+        u(x, t) = \cos(\pi x)\cos(t).
+
+    In this class the problem is implemented in the way that it is solved in space using FEniCS [1]_. Hence, the problem
```

Comment:
This formulation is a bit convoluted. Maybe `The spatial part is solved using FEniCS`?